src/stratis_cli/_actions/_logical.py

```diff
@@ -66,20 +70,26 @@ def list_volumes(namespace):
         proxy = get_object(TOP_OBJECT)
         managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
 
-        (pool_object_path, _) = unique(
-            pools(props={
-                'Name': namespace.pool_name
-            }).search(managed_objects))
-        matching_filesystems = filesystems(props={
-            'Pool': pool_object_path
-        }).search(managed_objects)
+        (properties, path_to_name) = get_pools(namespace, "pool_name",
+                                               managed_objects)
 
-        tables = [[
-            MOFilesystem(info).Name(),
-        ] for _, info in matching_filesystems]
+        mofilesystems = [
+            MOFilesystem(info) for _, info in filesystems(props=properties)
+            .search(managed_objects)
+        ]
 
-        print_table(['Name'], sorted(tables, key=lambda entry: entry[0]),
-                    ['<'])
+        tables = [[
+            path_to_name[mofilesystem.Pool()],
+            mofilesystem.Name(),
+            str(Range(mofilesystem.Used(), 1)),
```

Comment:
```str(Range(mofilesystem.Used()))``` is better. I think the "1" can not do any harm, but it should be entirely unnecessary.